test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -324,30 +324,42 @@ Describe "Import-Module from CompatiblePSEditions-checked paths" -Tag "CI" {
 }
 
 Describe "PSModulePath changes interacting with other PowerShell processes" -Tag "Feature" {
-    BeforeAll {
-        if (-not $IsWindows)
-        {
-            return
+    $PSDefaultParameterValues = @{ 'It:Skip' = (-not $IsWindows) }
+
+    Context "System32 module path prepended to PSModulePath" {
+        BeforeAll {
+            if (-not $IsWindows)
+            {
+                return
+            }
+            Add-ModulePath (Join-Path $env:windir "System32\WindowsPowerShell\v1.0\Modules") -Prepend
         }
-        Add-ModulePath (Join-Path $env:windir "System32\WindowsPowerShell\v1.0\Modules") -Prepend
-    }
 
-    AfterAll {
-        if (-not $IsWindows)
-        {
-            return
+        AfterAll {
+            if (-not $IsWindows)
+            {
+                return
+            }
+            Restore-ModulePath
+        }
+
+        It "Allows Windows PowerShell subprocesses to call `$PSHome modules still" -Skip:(-not $IsWindows) {
```

Comment:
`-Skip:(-not $IsWindows)` shouldn't be needed here as you have `$PSDefaultParameterValues` set at the beginning.